releases/README.md

```diff
@@ -0,0 +1,53 @@
+## containerd release process
+
+1. Create release pull request with release notes and updated versions.
+
+   1. Compile release notes detailing features added since the last release and
+      add release template file to `releases/` directory. The template is defined
+      by containerd's release tool but refer to previous release files for style
+      and format help. Name the file using the version, for rc add an `-rc` suffix.
+      When moving from rc to final, the rc file may just be renamed and updated.
+      See https://github.com/containerd/project/tree/master/cmd/release-tool
+
+   2. Update the version file at `https://github.com/containerd/containerd/blob/master/version/version.go`
+
+   3. Update RELEASES.md to refer to the new release and dates.
+
+   4. Update the `.mailmap` files for commit authors which have multiple email addresses in the commit log.
```

Comment:
It is a manual process today, normally I only have to do it during major releases but sometimes on patch releases when someone uses a new email somewhere. A line is just added for each duplicate email, which is just found manually. Sometimes this involves reaching out to the contributor to figure out how they want their name displayed in the release notes and which email they consider their main email. I will add some detail here.

Reply:
We should update the release tool to do this, and handle duplicates at the same time

Reply:
This might be beyond the capabilities of the release tool since recognizing similarities in names is not a process which is easy to automate. However if we get better about ensuring commit signoffs are done correctly (using real names), then we could possibly use that additional information to make manual intervention very rare.

Reply:
ok, so we need machine learning and language processing in our tool, check...